src/components/Profile/CoAuthoringTab.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import { CoauthorAttributes } from '../../entities/Coauthor/types'
+import usePaginatedProposals from '../../hooks/usePaginatedProposals'
+import Empty from '../Common/Empty'
+import FullWidthButton from '../Common/FullWidthButton'
+import SkeletonBars from '../Common/SkeletonBars'
+import Watermelon from '../Icon/Watermelon'
+
+import ProposalCreatedItem from './ProposalCreatedItem'
+
+interface Props {
+  address?: string
+  pendingCoauthorRequests?: CoauthorAttributes[]
+}
+
+const CoAuthoringTab = ({ address, pendingCoauthorRequests }: Props) => {
+  const [account] = useAuthContext()
+  const t = useFormatMessage()
+
+  const { proposals, hasMoreProposals, loadMore, isLoadingProposals } = usePaginatedProposals({
+    load: !!account,
+    ...((!!account || !!address) && { user: account || address }),
```

Comment:
you can extract this as you did in other components

`const user = address || account || undefined`